src/main/java/ee/carlrobert/codegpt/util/Utils.java

```diff
@@ -0,0 +1,15 @@
+package ee.carlrobert.codegpt.util;
+
+import java.util.Objects;
+import org.jetbrains.deft.Obj;
+
+public class Utils {
```

Comment:
The `com.intellij.openapi.util.text.StringUtil.equals` return `false` if a string is null and the other is empty, which would lead to `isModified` falsely returning `true` because empty TextFields always return an empty string and not `null`, but I fixed it at those places with `StringUtils.defaultString`